tests/functional/pages/surveys/census/communal/further-contact.page.js

```diff
@@ -1,4 +1,4 @@
-// >>> WARNING THIS PAGE WAS AUTO-GENERATED ON 2016-12-14 14:19:14.086947 - DO NOT EDIT!!! <<<
+// >>> WARNING THIS PAGE WAS AUTO-GENERATED - DO NOT EDIT!!! <<<
```

Comment:
I'd say given that quite a few of the files needed to be re-generated, it makes sense to remove the timestamp in this pull request. The other changes are small and easily identified.